src/Spice86.Core/Emulator/VM/Machine.cs

```diff
@@ -207,132 +195,12 @@ public sealed class Machine : IDisposable, IDebuggableComponent {
-        VideoInt10Handler = new VgaBios(Memory, Cpu, VgaFunctions, BiosDataArea, loggerService);
-
-        TimerInt8Handler = new TimerInt8Handler(Memory, Cpu, DualPic, Timer, BiosDataArea, loggerService);
-        BiosKeyboardInt9Handler = new BiosKeyboardInt9Handler(Memory, Cpu, DualPic, Keyboard, BiosDataArea, loggerService);
-
-        BiosEquipmentDeterminationInt11Handler = new BiosEquipmentDeterminationInt11Handler(Memory, Cpu, loggerService);
-        SystemBiosInt12Handler = new SystemBiosInt12Handler(Memory, Cpu, BiosDataArea, loggerService);
-        SystemBiosInt15Handler = new SystemBiosInt15Handler(Memory, Cpu, Memory.A20Gate, loggerService);
-        KeyboardInt16Handler = new KeyboardInt16Handler(Memory, Cpu, loggerService, BiosKeyboardInt9Handler.BiosKeyboardBuffer);
-
-        SystemClockInt1AHandler = new SystemClockInt1AHandler(Memory, Cpu, loggerService, TimerInt8Handler);
-
-        MouseDriver = new MouseDriver(Cpu, Memory, MouseDevice, gui, VgaFunctions, loggerService);
-        Dos = new Dos(Memory, Cpu, KeyboardInt16Handler, VgaFunctions, configuration.CDrive, configuration.Exe, loggerService);
-
-        if (configuration.InitializeDOS is not false) {
-            // Register the interrupt handlers
-            RegisterInterruptHandler(VideoInt10Handler);
-            RegisterInterruptHandler(TimerInt8Handler);
-            RegisterInterruptHandler(BiosKeyboardInt9Handler);
-            RegisterInterruptHandler(BiosEquipmentDeterminationInt11Handler);
-            RegisterInterruptHandler(SystemBiosInt12Handler);
-            RegisterInterruptHandler(SystemBiosInt15Handler);
-            RegisterInterruptHandler(KeyboardInt16Handler);
-            RegisterInterruptHandler(SystemClockInt1AHandler);
-            RegisterInterruptHandler(Dos.DosInt20Handler);
-            RegisterInterruptHandler(Dos.DosInt21Handler);
-            RegisterInterruptHandler(Dos.DosInt28Handler);
-            RegisterInterruptHandler(Dos.DosInt2FHandler);
-
-            // Initialize DOS.
-            Dos.Initialize(SoundBlaster, CpuState, configuration.Ems);
-            if (Dos.Ems is not null) {
-                RegisterInterruptHandler(Dos.Ems);
-            }
-
-            var mouseInt33Handler = new MouseInt33Handler(Memory, Cpu, loggerService, MouseDriver);
-            RegisterInterruptHandler(mouseInt33Handler);
-
-            var mouseIrq12Handler = new BiosMouseInt74Handler(DualPic, Memory);
-            RegisterInterruptHandler(mouseIrq12Handler);
-
-            SegmentedAddress mouseDriverAddress = AssemblyRoutineInstaller.InstallAssemblyRoutine(MouseDriver);
-            mouseIrq12Handler.SetMouseDriverAddress(mouseDriverAddress);
-        }
-    }
-
+    
     /// <summary>
-    /// The pause handler for the entire machine.
+    /// The size of the conventional memory in kilobytes.
     /// </summary>
-    public IPauseHandler PauseHandler { get; }
-
```

Comment:
How am I supposed to pause the machine from my CSharpOverrides now?

Reply:
Should not it be a public property in the CSharpOverrideHelper class ?

I'll revert it too anyway. This change is hard to justify since the Machine will be kept after all.

Reply:
Reverted.